src/ContractInteractor.ts

```diff
@@ -145,12 +150,18 @@ export default class ContractInteractor {
             contractName: 'ITokenHandler',
             abi: ITokenHandler.abi
         });
+        // @ts-ignore
```

Comment:
What is this `@ts-ignore` for? I'd add a comment explaining why do we need it if we cannot remove it.